### DISCUSSION TITLE
Setting the guitab font family and size

### DISCUSSION THREAD

**Discussion by ir0nh34d** [macvim-dev/macvim#1421] [General]
Is it possible to set the font family and size of the guitab in MacVim?

**Comment by ychin** [comment]
Not right now. Why would you want to change the font btw? It should be using the font similar to what the OS is using.

**Comment by ir0nh34d** [comment]
I find the font in the tab bar to be small.  I've set guifont so that the window font works for me and the Window titlebar and status bar all have an appropriate font.
